rest_framework_smoke/tests/mixins.py

```diff
@@ -145,7 +145,7 @@ def get_list_schema(self, min_items: int = 1) -> dict:
         """ Returns list response schema with pagination data
         """
         result_list_schema = {
-            "type": "array",
+            "type": ["array"],
```

Comment:
Почему тогда тесты не падали?

Reply:
Потому что тесты падают только если есть пагинация. А в ProjectViewSetTestCase она отключена
```
pagination_schema = None
```
Если быть точнее, то в тестах получается вот такое:
```
jsonschema.validate(obj, {"type": "array", ...})
```
А при включении пагинации получаем вот такое:
```
jsonschema.validate(obj, schemas.get_object_schema({"type": "array", ...}))
```
Ну а внутри `get_object_schema` следующая строчка делает из строк список
```
        # remove null because of high probability of type check skip
        attribute["type"] = [t for t in attribute["type"] if t != "null"]
```

Reply:
В итоге тесты падают с ошибкой 
```jsonschema.exceptions.SchemaError: ['a', 'r', 'r', 'a', 'y'] is not valid under any of the given schemas```

Reply:
Значит нужен тест с пагинацией, который падал бы без этого исправления.

Reply:
Поправил тест списка на работу с пагинацией